Microsoft.Toolkit.Uwp.UI.Controls/BladeView/BladeView.xaml

```diff
@@ -98,42 +98,47 @@
         <Setter Property="Margin" Value="0" />
         <Setter Property="MinWidth" Value="{ThemeResource GridViewItemMinWidth}" />
         <Setter Property="MinHeight" Value="{ThemeResource GridViewItemMinHeight}" />
-        <Setter Property="BorderBrush" Value="Black" />
+        <Setter Property="BorderBrush" Value="{ThemeResource SystemChromeHighColor}" />
         <Setter Property="BorderThickness" Value="1" />
         <Setter Property="IsTabStop" Value="False" />
         <Setter Property="IsExpanded" Value="True"/>
+        <Setter Property="Width" Value="320" />
         <Setter Property="HeaderTemplate">
             <Setter.Value>
                 <DataTemplate>
-                    <TextBlock Text="{Binding}" Margin="4,0,0,0" VerticalAlignment="Center"/>
+                    <TextBlock 
+                        Text="{Binding}" 
+                        Margin="12,7,0,9" 
+                        VerticalAlignment="Center" />
                 </DataTemplate>
             </Setter.Value>
         </Setter>
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="controls:BladeItem">
                     <Grid BorderBrush="{TemplateBinding BorderBrush}"
-                          BorderThickness="{TemplateBinding BorderThickness}">
+                          BorderThickness="{TemplateBinding BorderThickness}"
+                          Width="{TemplateBinding Width}">
```

Comment:
It will still collapse. 

Although, I noticed in my version that it would collapse down to only the width of the 2 buttons, but after merging in the rel/3.0.0-preview, the functionality changed to include the header as well. I'm not actually sure what the expected interaction is though. 

@nmetulev, should the collapsed mode include the buttons AND header text? It does currently, but I just want to be sure that is expected.

Reply:
IMO, it should collapse the same way it currently collapses.

Reply:
Sounds good